src/sailing_robot/tests/test_angle_average.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from sailing_robot.navigation import angle_average
+
+class AngleAverageTests(unittest.TestCase):
+    def realValues(self):
```

Comment:
double check that these are actually running - I think test methods may need to be called `test_*`